helm/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: monokle-admission-controller-server
+  namespace: {{ include "monokle-admission-controller.namespace" . }}
+  labels:
+    app: monokle-admission-controller-server
+    {{- include "monokle-admission-controller.labels" . | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
This should at some point be configurable and maybe even allow autoscaling.

That's not to ensure load capacity but availability. Take for example our staging & dev clusters where the nodes restart every couple of ours and so. During these kinds of periods, a single instance might become unavailable and we would not want that.

Maybe we could get some actual usage of the controller first, but for future record.

Reply:
Good point. I was thinking about this for a moment. I can make it helm variable as this should be a quick one.

Reply:
Add configurable replicas here - https://github.com/kubeshop/monokle-admission-controller/pull/8/commits/979d087a8b610e9d3cc60f62a61b2abf1b3faa9b.

Should I report an issue regarding autoscaling? 🤔

Reply:
Reported as https://github.com/kubeshop/monokle-admission-controller/issues/11